android/KMEA/app/src/main/java/com/tavultesoft/kmea/ModelPickerActivity.java

```diff
@@ -205,6 +205,8 @@ public void onBackPressed() {
 
   @Override
   protected void onActivityResult(int requestCode, int resultCode, Intent data) {
+    super.onActivityResult(requestCode, resultCode, data);
+
     if (resultCode == 1) {
```

Comment:
It seems, that we never get a resultCode=1 here.
Maybe the it is a bug, and we need to check the request Code?

This would make the model picker, closing when closing the model info.

Reply:
maybe this unintentionally works.

When closing the model info, we want to remain on ModelPickerActivity for the user to decide what to do next.

Reply:
Maybe we want to delete the code, when it is never used?